client/src/test/java/io/pravega/client/CredentialsExtractorTest.java

```diff
@@ -10,83 +10,202 @@
 package io.pravega.client;
 
 import io.pravega.client.stream.impl.Credentials;
+
 import java.util.HashMap;
+import java.util.Map;
 import java.util.Properties;
-import org.junit.Assert;
 import org.junit.Test;
 
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
 
 public class CredentialsExtractorTest {
-    @Test
-    public void testextractCredentials() {
-        //No creds defined
-        ClientConfig config = ClientConfig.builder().build();
-        assertEquals("Empty list should return null", config.getCredentials(), null);
 
-        //Test custom creds
+    @Test
+    public void extractsCredentialsFromProperties() {
```

Comment:
For uniformity, can we start the test method name with `test`?

Reply:
I've added the `test` prefix to all the test methods now.